### PR TITLE
Add hammer_cli_foreman_openscap 0.1.1

### PR DESCRIPTION
**Pull request by xprazak2**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_foreman_openscap
+%global confdir hammer
+
+Summary: Foreman Openscap commands for Hammer CLI
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.1
+Release: 2%{?foremandist}%{?dist}
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec]: Start at `1`

File: rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Group: Applications/System
+License: GPLv3
+URL: http://github.com/theforeman/hammer_cli_foreman_openscap
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.1.3
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec]: The gem dependency states `>= 0.6.0`.

File: rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec [hunk 3]
```diff
@@ -0,0 +1,68 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_foreman_openscap
+%global confdir hammer
+
+Summary: Foreman Openscap commands for Hammer CLI
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec]: Should be "OpenSCAP" - note that this doesn't match the summary in the gemspec, they ought to be the same.

**Comment by xprazak2** [comment]
The description in gemspec will be improved in the next release, is that ok?

**Comment by domcleal** [comment]
Sure, that's fine. Merged and building for nightly only, thanks @xprazak2.

**Comment by ares** [comment]
Could we please get this also tagged in 1.13? I just tested it with `foreman-1.13.0-0.develop.201609051301git713be41.el7.noarch` and it seems to work fine.

**Comment by domcleal** [comment]
Cherry-picked as bdee306 to rpm/1.13 and building.

**Comment by ares** [comment]
thank you!

**Pull request merged**